src/Rhone/JS/Sink.idr

```diff
@@ -235,3 +235,22 @@ namespace LocalStorage
   export %inline
   setItemAt : LiftJSIO m => (key : String) -> MSF m String ()
   setItemAt = firstArg setItem
+
+--------------------------------------------------------------------------------
+--          Focus
+--------------------------------------------------------------------------------
+
+export
+setFocus : LiftJSIO m => (0 _ : JSType t) => (0 _ : Elem HTMLOrSVGElement (Types t)) => SafeCast t =>
```

Comment:
Would you mind breaking up these lengthy lines to not exceed a width of 80 characters? I often work with split screens an am therefore glad if the code I work with does not exceed this limit.

(Note to self: I probably should read up on how to add such a rule to the linter used by the the CI)

Reply:
Sure. Done!